lms/djangoapps/course_blocks/api.py

```diff
@@ -0,0 +1,57 @@
+"""
+...
+"""
+from django.core.cache import cache
+
+from openedx.core.lib.block_cache.block_cache import get_blocks, clear_block_cache
+from xmodule.modulestore.django import modulestore
+
+from .transformers import (
+    library_content,
+    split_test,
+    start_date,
+    user_partitions,
+    visibility,
+)
+from .user_info import CourseUserInfo
+
+
+LMS_COURSE_TRANSFORMERS = [
+    library_content.ContentLibraryTransformer(),
+    split_test.SplitTestTransformer(),
+    start_date.StartDateTransformer(),
+    user_partitions.UserPartitionTransformer(),
+    visibility.VisibilityTransformer(),
+]
+
+
+def _get_cache():
+    """Function exists for mocking/testing, or if we want a custom cache."""
+    return cache
+
+
+def get_course_blocks(
+        user,
+        root_usage_key=None,
+        course_key=None,
+        transformers=None
+):
+    """
+    TODO
+    """
```

Comment:
Probably better to leave out docstring stubs like this, so pylint can catch it for us.